workflow/dependency_injector_demo.py

```diff
@@ -0,0 +1,55 @@
+from typing import reveal_type
```

Comment:
is this file part of the workflow or is it something you're just testing out? Maybe we could just drop it for clarity?